extensions/key-value/package.json

```diff
@@ -28,15 +28,46 @@
       "title": "Find Value by Key",
       "description": "Shows a list of key-value pairs from a JSON file and allows you to copy them to the clipboard",
       "mode": "view"
+    },
+    {
+      "name": "reload-url-no-view",
+      "title": "Reload from URL",
+      "description": "Reloads the JSON file from the URL",
+      "mode": "view",
+      "preferences": [
+        {
+          "name": "jsonURLPath",
+          "description": "URL path to a json file with key value pairs",
+          "type": "textfield",
+          "required": true,
+          "title": "Key Value Json URL"
+        },
+        {
+          "name": "jsonKeyForKey",
+          "description": "Key name to search for",
+          "type": "textfield",
+          "required": true,
+          "title": "Key"
+        },
+        {
+          "name": "jsonKeyForValue",
+          "description": "Key name to get the value from",
+          "type": "textfield",
+          "required": true,
+          "title": "Value"
+        }
+      ]
```

Comment:
Should the new preferences be required?

Reply:
It's should be required only under the reload command

Reply:
Makes sense 🙂